packages/shared/src/hooks/streaks/useStreakRecover.ts

```diff
@@ -58,13 +60,18 @@ export const useStreakRecover = ({
         .request(USER_STREAK_RECOVER_MUTATION)
         .then((res) => res.recoverStreak),
     onSuccess: () => {
-      displayToast('Lucky you! Your streak has been restored');
       logEvent({
         event_name: LogEvent.StreakRecover,
       });
     },
   });
 
+  const hideRemoteAlert = useCallback(async () => {
```

Comment:
I think it doesn't have to be at the close function. At the mount of the popup, we should send a mutation. The pop-up will stay open anyway even after that. Thoughts? cc @dailydotdev/web-team

Reply:
Yes, I put on close because maybe using extension you open a new tab but don't even read what's inside. 
Maybe if you close you had the chance!

Reply:
Yeah, makes sense.